packages/eslint-plugin/tests/rules/no-misused-spread.test.ts

```diff
@@ -741,6 +741,17 @@ ruleTester.run('no-misused-spread', rule, {
           endLine: 6,
           line: 3,
           messageId: 'noMapSpreadInObject',
+          suggestions: [
+            {
+              messageId: 'replaceMapSpreadInObject',
+              output: `
+        const o = Object.fromEntries(new Map([
+            ['test-1', 1],
+            ['test-2', 2],
+          ]));
+      `,
+            },
+          ],
```

Comment:
[Testing] Just to be thorough, could you add another case like this with `...(map)` instead of `...map`? It works right now, but parenthesis tend to be an edge case we break on over time.